Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:alpine
+WORKDIR /Rest-api
+COPY requirements.txt makefile student.py /Rest-api/ 
```

Comment:
Make a note to yourself to read and understand about docker layer catching optimization tricks to reduce the docker build time. 
This is a standard practice and you will find enough material online to read.